From aaf4d1f40be0858c34722a6d8f4e0a4f6ca44f41 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Mon, 4 Dec 2017 10:12:34 +0800
Subject: [PATCH 2967/5242] MLK-15985-2: ARM: dts: imx6qdl: set freq for
 S20_3LE format

commit  84cd7b854955a0e4b4f94ef8f184e97862b05d00 from
https://source.codeaurora.org/external/imx/linux-imx.git

select a proper freq for S20_3LE/S24_LE/S16_LE, the frequency
can be divided by 24/32/16;

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index fd3a563..9b4f4b6 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -1104,7 +1104,7 @@
 	assigned-clock-parents = <&clks IMX6QDL_CLK_OSC>,
 				 <&clks IMX6QDL_CLK_PLL4>,
 				 <&clks IMX6QDL_CLK_PLL4_AUDIO_DIV>;
-	assigned-clock-rates = <1179648000>, <0>, <0>;
+	assigned-clock-rates = <737280000>, <0>, <0>;
 	status = "okay";
 };
 
-- 
1.7.9.5

